### PR TITLE
Package renamed to tableschema

### LINKED ISSUE #148: Rename to tableschema
# Overview

Should use name `tableschema`.

# Tasks

- [ ] update name to `tableschema`

### PR DESCRIPTION
**Pull request by roll**
- fixes #148

### CODE REVIEW

**Comment by roll** [comment]
@pwalsh 
could you please review?

File: README.md [hunk 1]
```diff
@@ -111,7 +119,7 @@ Where the option `no_fail_fast` is given, it will collect all errors it encouter
 ### Field
 
 ```python
-from jsontableschemal import Field
+from tableschemal import Field
```
Comment (pwalsh) [inline README.md]: ?

File: tableschema/model.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def _expand(self, schema):
 
     @staticmethod
     def _type_map():
-        """Map a JSON Table Schema type to a JTSKit type class."""
+        """Map a Table Schema type to a JTSKit type class."""
```
Comment (pwalsh) [inline tableschema/model.py]: Seeing as you started renaming first :), this still has the very first name - JTSKit.

File: README.md (hunk 1 above)
Comment (roll) [inline README.md]: Thanks. Fixed.

**Pull request merged**